src/common/utils.ts

```diff
@@ -539,7 +539,7 @@ export class URIUtils {
             }
             return retParams;
         }
-        const toPath = compile(pathTemplate, { encode: encodeURIComponent });
+        const toPath = compile(pathTemplate);
```

Comment:
question: why do yo need to update this?

Reply:
The previous problem was to solve the problem that the generated URL contains multiple characters: / etc. now it is no longer needed. I will change it back

Reply:
got it, thanks!